homeassistant/components/harmony/remote.py

```diff
@@ -110,44 +111,19 @@ async def async_setup_entry(
     _LOGGER.debug("Harmony Remote: %s", device)
 
     async_add_entities([device])
-    register_services(hass)
+    register_services()
```

Comment:
Why not move setting up those 2 services directly into async_setup_entry?
To match it up with other integrations a bit more?

Other than that 👍

Reply:
@frenck A few less lines now... and done!